### DISCUSSION TITLE
Self-nomination for Tooling Contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#163] [Tooling Team]
Hi @paketo-buildpacks/tooling-maintainers 

I would like to self-nominate for the Contributor role for the tooling team. I have made a variety of commits to and code-reviews for `occam`, and I am actively interested in continuing to contribute.

Thanks,
Rob

**Comment by robdimsdale** [comment]
@fg-j @sophiewigmore can I get one more 👍 so we have a supermajority please?

**Comment by robdimsdale** [comment]
https://github.com/orgs/paketo-buildpacks/teams/steering-committee looks like we have a super majority for me to be a member of the tooling contributors team.

**Comment by dmikusa** [comment]
Congrats! I have added you to the contributor's team.
